problems/euler/num22-names-scores/ruby/DanielBachler/solution.rb

```diff
@@ -0,0 +1,26 @@
+# Enter your code here. Read input from STDIN. Print output to STDOUT
```

Comment:
Maybe elaborate on this comment with what you had in the readme, with things like enter an integer, followed by that many names (was a little confused at first trying to immediately figure out what to do.